requirements/prod_common.txt

```diff
@@ -364,6 +364,9 @@ pygit2==0.27.4 \
 webencodings==0.5.1 \
     --hash=sha256:a0af1213f3c2226497a97e2b3aa01a7e4bee4f403f95be16fc9acd2947514a78 \
     --hash=sha256:b36a1c245f2d304965eb4e0a82848379241dc04b865afcc4aab16748587e1923
-premailer==3.2.0 \
-    --hash=sha256:a344b2013f8e099962bdea3a433fbea8614006fbe2ef62cd89c653a2b33290ad \
-    --hash=sha256:ca97cec6115fea6590b49558c55d891996f9eb4da6490c7b60c3a8af4c8c0735
+premailer==3.3.0 \
+    --hash=sha256:f45eb4a30485aeccc3ff19771d6614346899ec19a301931af4694f737b6035c3 \
+    --hash=sha256:93be4f197e9d2a87a8fe6b5b6a79b64070dbb523108dfaf2a415b4558fc78ec1
+cachetools==3.1.0 \
+    --hash=sha256:219b7dc6024195b6f2bc3d3f884d1fef458745cd323b04165378622dcc823852 \
+    --hash=sha256:9efcc9fab3b49ab833475702b55edd5ae07af1af7a4c627678980b45e459c460
```

Comment:
Missing empty line

Reply:
I can add it, but the parent revision doesn't have one either.

Reply:
Done in https://github.com/mozilla/addons-server/pull/10644/commits/00b80e7ed62cd342f5a97af9024884fc20bf4be8